solc/CommandLineInterface.cpp

```diff
@@ -664,31 +664,40 @@ bool CommandLineInterface::parseLibraryOption(string const& _input)
 	for (string const& lib: libraries)
 		if (!lib.empty())
 		{
-			//search for last colon in string as our binaries output placeholders in the form of file:Name
-			//so we need to search for the second `:` in the string
-			auto colon = lib.rfind(':');
-			if (colon == string::npos)
+			//search for equal sign in string as our binaries output placeholders in the form of file=Name
+			//so we need to search for `=` in the string
+			auto equalSign = lib.find('=');
```

Comment:
Why the change from `rfind` to `find`? I guess before it was `rfind` exactly because of the double `:` problem. Maybe we should `solAssert(lib.rfind('=') == lib.find('='), "");`?

Reply:
The reason I changed ```rfind``` to ```find``` is I think there is only one "="  in the string and ```find``` is clearer for developer.

But ```rfind``` is OK to me. I will rollback.

Do we still need ```solAssert(lib.rfind('=') == lib.find('='), "Only one equal sign \"=\" is allowed in the address string.");``` here ?

Reply:
I'm fine with `find` instead of `rfind`. Maybe I'm just being too paranoid with the assertion

Reply:
Sure. I have added ```solAssert``` here.